extensions/wikia/AdMixExperiment/scripts/AdMixExperiment.js

```diff
@@ -0,0 +1,340 @@
+require(['ext.wikia.adEngine.adContext', 'wikia.abTest', 'wikia.throttle'], function (adContext, abTest, throttle) {
+	$(function () {
+		var context = adContext.getContext();
+		var $adAndRecircWrapper;
+		var $topRightAd = $('#TOP_RIGHT_BOXAD');
+		var $floatingMedrec = $('#INCONTENT_BOXAD_1');
+		var $topRightAdWrapper = $('#top-right-boxad-wrapper');
+		var $footer = $('#WikiaFooter');
+		var $rail = $('#WikiaRail');
+		var $recirc;
+		var $recircWrapper;
+		var $visibleElementBeforeWrapper;
+		var $window = $(window);
+
+		var adMixRecircWrapperHeight = 0;
+		var bottomMargin = 20;
+		var breakpointSmall = 1023;
+		var globalNavigationHeight = $('#globalNavigation').outerHeight(true);
+		var gapSize = 200;
+		var firstAdTopSpace = 10;
+		var noRecircAdBottomSpace = 30;
+		var recircTopSpace = 10;
+		var topRightAdHeight = 250;
+		var viewportHeight;
+		var viewportWidth;
+		var visibleElementBeforeWrapperHeight;
+
+		// ad mix flags
+		// AD_MIX_1 & AD_MIX_1B: reloadFloatingMedrec === true && recircEnabled === true
+		// AD_MIX_2 & AD_MIX_2B: topRightAdFixed === true && recircEnabled === true
+		// AD_MIX_3 & AD_MIX_3B: reloadRecirc === true && recircEnabled === true
+		// CONTROL: all === false
+		var reloadFloatingMedrec = abTest.getGroup('AD_MIX') === 'AD_MIX_1' || abTest.getGroup('AD_MIX') === 'AD_MIX_1B';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'reloadFloatingMedrec' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "reloadFloatingMedrec" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)